core/imageroot/var/lib/nethserver/cluster/bin/print-inventory-nsent

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
```

Comment:
Why 2 different inventory scripts? One should be enough for both remote servers.